Content.Server/_DV/CosmicCult/Components/CleanseOnUseComponent.cs

```diff
@@ -0,0 +1,36 @@
+using Content.Shared.Damage;
+using Robust.Shared.Audio;
+using Robust.Shared.Prototypes;
+
+namespace Content.Server._DV.CosmicCult.Components;
+
+[RegisterComponent, AutoGenerateComponentState]
+[Access(typeof(DeconversionSystem))]
+public sealed partial class CleanseOnUseComponent : Component
+{
+    [DataField] public TimeSpan UseTime = TimeSpan.FromSeconds(10);
+
+    [DataField] public SoundSpecifier SizzleSound = new SoundPathSpecifier("/Audio/Effects/lightburn.ogg");
+
+    [DataField] public SoundSpecifier CleanseSound = new SoundPathSpecifier("/Audio/_DV/CosmicCult/cleanse_deconversion.ogg");
+    [DataField] public SoundSpecifier MalignSound = new SoundPathSpecifier("/Audio/_DV/CosmicCult/glyph_trigger.ogg");
```

Comment:
Add blank line for consistency
```suggestion
    [DataField] public SoundSpecifier CleanseSound = new SoundPathSpecifier("/Audio/_DV/CosmicCult/cleanse_deconversion.ogg");

    [DataField] public SoundSpecifier MalignSound = new SoundPathSpecifier("/Audio/_DV/CosmicCult/glyph_trigger.ogg");
```